vowpalwabbit/parse_args.cc

```diff
@@ -1144,6 +1145,7 @@ void parse_reductions(vw& all)
   all.reduction_stack.push_back(bs_setup);
 
   all.reduction_stack.push_back(audit_regressor_setup);
+  all.reduction_stack.push_back(classweight_setup);
```

Comment:
This is to late in the reduction stack.   Can you shift to the spot I pointed out?